server/restapi/tests.py

```diff
@@ -44,3 +46,58 @@ def test_user_can_send_message(self):
         self.assertEqual(senderResponse.status_code, 201)
         self.assertEqual(senderResponse.json()['text'],
                          senderRequest['text'])
+
+
+class MessageListTestCase(TestCase):
+    """User can get message list"""
+
+    def test_user_can_get_message_list(self):
+        c = Client()
+        # Signup sender test user
+        users_id = []
+        USER_COUNT = 3
+        for i in range(USER_COUNT):
+            senderRequest = {'name': 'username_' + str(i)}
+            senderResponse = c.post('/api/signup', senderRequest)
+            user_id = senderResponse.json()['id']
+            users_id.append(user_id)
```

Comment:
```suggestion
          user1 = fake_user("username_1")
          user2 = fake_user("username_2")
```

Воспользуйся здесь extract method, чтобы не дублировать код, только в тесте на создание нужно явно написать. https://refactoring.guru/ru/extract-method